notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -53,30 +43,10 @@ private class NotifiqueListView(
     })
   }
 
-  @WorkerThread
-  private fun pagedList(sourceFactory: DataSource.Factory<Int, Notifique>): PagedList<Notifique> {
-    val source = sourceFactory.create()
-    source.addInvalidatedCallback {
-      val list = pagedList(sourceFactory)
-      launch(UI) { listAdapter.submitList(list) }
-    }
-    return PagedList.Builder(source, 20)
-        .setMainThreadExecutor(MainThreadExecutor)
-        .setBackgroundThreadExecutor(BackgroundThreadExecutor)
-        .build()
-  }
-
   override fun onAttachedToWindow() {
     super.onAttachedToWindow()
-    databaseJob = launch(UI) {
-      val list = async { pagedList(dao.sourceFactory()) }.await()
-      listAdapter.submitList(list)
-    }
-  }
-
-  override fun onDetachedFromWindow() {
-    super.onDetachedFromWindow()
-    databaseJob.cancel()
+    LivePagedListBuilder(dao.sourceFactory(), 20).build()
+        .observeForever { listAdapter.submitList(it) }
```

Comment:
Discussed elsewhere, and it's apparently not guaranteed that the LiveData won't be held onto globally by the library.
Uuuuugh wut.

Will just throw in a mirroring unregister call until we decide to roll this ourselves.